docs/mm_quick_start.md

```diff
@@ -0,0 +1,181 @@
+
+```
+git init && git remote add -f origin https://github.com/triton-inference-server/model_analyzer.git
+```
+
+**3. Enable sparse checkout, and download the examples directory, which contains the add_sub & resnet50_libtorch models**
+
+```
+git config core.sparseCheckout true && \
+echo 'examples' >> .git/info/sparse-checkout && \
+git pull origin main
+```
+
+## `Step 2:` Pull and Run the SDK Container
+
+---
+
+**1. Pull the SDK container:**
+
+```
+docker pull nvcr.io/nvidia/tritonserver:23.01-py3-sdk
+```
+
+**2. Run the SDK container**
+
+```
+docker run -it --gpus all \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      -v $(pwd)/examples/quick-start:$(pwd)/examples/quick-start \
+      -v <path-to-output-model-repo>:<path-to-output-model-repo> \
+      --net=host nvcr.io/nvidia/tritonserver:23.01-py3-sdk
+```
+
+**Replacing** `<path-to-output-model-repo>` with the
+**_absolute_ _path_** to the directory where the output model repository
+will be located.
+This ensures the Triton SDK container has access to the model
+config variants that Model Analyzer creates.<br><br>
+**Important:** You must ensure the absolutes paths are identical on both sides of the mounts<br><br>
+
+## `Step 3:` Profile both models concurrently
+
+---
+
+The [examples/quick-start](../examples/quick-start) directory is an example
+[Triton Model Repository](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_repository.md) that contains two libtorch models: `add_sub` & `resnet50_libtorch`
+
+Run the Model Analyzer `profile` subcommand inside the container with:
+
+```
+model-analyzer profile \
+    --model-repository <path-to-examples-quick-start> \
+    --profile-models add_sub,resnet50_libtorch --triton-launch-mode=docker \
+    --output-model-repository-path <path-to-output-model-repo>/<output_dir>
+    --export-path profile_results
+    --run-config-profile-models-concurrently-enable
+```
+
+**Important:** You must specify an `<output_dir>` subdirectory. You cannot have `--output-model-repository-path` point directly to `<path-to-output-model-repo>`
+
+**Important:** If you already ran this earlier in the container, you can use the `--override-output-model-repository` option to overwrite the earlier results.
+
+**Important**: All models must be in the same directory
+
+**Important:** The checkpoint directory should be removed between consecutive runs of
+the `model-analyzer profile` command.<br><br>
+
+---
+
+The `--run-config-profile-models-concurrently-enable` option tells MA to load and optimize both models concurrently using the [Quick Search](config_search.md#quick-search-mode) algortihm.
+
+This will profile both models concurrently using the quick search algorithm to find the maximal throughput gain for both models by iterating across instance counts and batch sizes. By default, the algorithm is attempting to find the best balance of gains for each model, not the best combined total throughput.
```

Comment:
you just said "using quick search algorithm" in the previous sentence

Reply:
wrong tense in "is attempting".  Should be "will attempt"

Reply:
Fixed wording & removed quick search